### DISCUSSION TITLE
Starting display, basics

### DISCUSSION THREAD

**Discussion by uhorkovyprivarok** [RudolphRiedel/FT800-FT813#40] [Q&A]
Hello, I heve problem, and need help. 

I have this display - GEN4-FT813-50CTP-CLB
link: https://4dsystems.com.au/products/4d-intelligent-hmi-display-modules/gen4-ftdi-display-modules/gen4-ft813-50ctp-clb

and I am using microcontroller TMS320F28335, and I use program Code Composer.

I use example which is here on github, but it doesn´t work. 

The question is if the display need some special initialization to light up, or I just need to power it and the display should light up. 

I am using SPI communication. Sending this:

#include <DSP2833x_Device.h>
#include "F28335_DeviceBasic.h"
#include "mcu.h"
#include "gpio.h"
#include "interrupts.h"
#include "spi_func.h"

#include "EVE_commands.h"
#include "EVE_config.h"
#include "EVE_target.h"
#include "EVE.h"

int main(void)
{
    DeviceInitialization();

    gpio_init();

    mcu_initSpia(500000); // 500kHz

    EVE_cmd_dl(CMD_DLSTART); // tells EVE to start a new display-list
    EVE_cmd_dl(DL_CLEAR_RGB | 0xFFFFFF); // sets the background color
    EVE_cmd_dl(DL_CLEAR | CLR_COL | CLR_STN | CLR_TAG);
    EVE_color_rgb(0x0);
    EVE_cmd_text(5, 15, 28, 0, "Hello there!");
    EVE_cmd_dl(DL_DISPLAY); // put in the display list to mark its end
    EVE_cmd_dl(CMD_SWAP); // tell EVE to use the new display list
    while (EVE_busy());

	return 0;
}

But the display don´t light or do something.

Thanks for respond.

**Comment by RudolphRiedel** [comment]
It has been a while since I used CCS, I just downloaded CCS 11.
First off I did not implement support for TMS320F28335 / C2000, yet, so in order to make it compile you must have added a couple more lines. Please share the project file so I can have a look at it.

And then, yes, the first thing to call is EVE_init().
I just started to explain the examples in more detail, this is not finished however.

**Reply by uhorkovyprivarok** [reply]
Sorry but I am new on github, but I guess I have uploadet files successfully on my profile and also added you as collaborator so you should see it.

**Reply by RudolphRiedel** [reply]
It's not like I know git inside-out and an invite for a colaboration is a first for me as well. :-)
I have a look.
I played a little with CCS 11 so far and did not get very far as I could not even find a led-blink example.
And I checked what kind of devkit TI might offer for the TMS320F28335 and only found the TMDSPREX28335 - at 177 EUR without taxes and shipping - so please excuse me when I give this a pass. :-)

**Comment by RudolphRrr** [comment]
I have no access to the other repository right now to comment there.
The issue with the logic analyzer might require some tinkering with Zadig: https://zadig.akeo.ie/
I just found this thread https://sigrok.org/bugzilla/show_bug.cgi?id=650 and in comment #25 it says:
"Finally got it working on all my Windows 7/10 Systems.
This is what i did, with Zadiag :
Install libusb-Driver, re-connect Device, install winUSB-Driver, re-connect Device, install libusbK, re-connect Device, then again installed libusb. 
And voila, started Pulsview and all OK."

I might have done that as well when I bought one of these cheap clone logic analyzer a while ago.
My experience with these cheap clones and Pulseview is limited as I have an original Saleae Logic 16 at my disposal.

**Reply by uhorkovyprivarok** [reply]
Hello Rudolph, thanks, it helps I guess.. I am not sure because I checked it very fastly, because now I have problem because I reinstalled with Zadig my laptom camera driver for winUSB-Driver.. :D any suggestions how to fix it? XD 

And if you tell me how to create discussion in my repository I will mage discussion, or if you want to connect to our team I can add you as colaborator.. :P Idk what is better.. 

But really thx for help with logic analyzer.. looks that it works..

**Reply by RudolphRiedel** [reply]
Looks like discussions are more a beta feature than a regular one, it popped up over here and I activated it.
Anyways, the other account up there is from work and shockingly I am supposed to work when I am at work. :-)
I just had a few minutes to spare while a test was running and looking into the logic analyzer issue felt like something worthwhile to do since I occasionally recommend these cheap clone LAs to colleagues, students or apprentices.
